src/MICore/project.json

```diff
@@ -32,7 +34,8 @@
     "System.Threading.Thread": "4.0.0",
     "System.Threading.Timer": "4.0.1",
     "System.Xml.ReaderWriter": "4.0.11",
-    "System.Xml.XmlSerializer": "4.0.11"
+    "System.Xml.XmlSerializer": "4.0.11",
+    "Newtonsoft.Json": "9.0.1"
```

Comment:
Looking at https://github.com/JamesNK/Newtonsoft.Json/releases I didn't see anything that blocks us from compiling against the older version. But I could be missing something.